sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/aggregator/MetricDataUtils.java

```diff
@@ -44,4 +45,23 @@ private MetricDataUtils() {}
             points.add(aggregator.toPoint(startEpochNanos, epochNanos, labels)));
     return points;
   }
+
+  static List<DoubleHistogramPointData> toDoubleHistogramPointList(
+      Map<Labels, HistogramAccumulation> accumulationMap,
+      long startEpochNanos,
+      long epochNanos,
+      List<Double> boundaries) {
+    List<DoubleHistogramPointData> points = new ArrayList<>(accumulationMap.size());
+    accumulationMap.forEach(
```

Comment:
As an exercise can you write this without lambda :)))

Reply:
seems that all `MetricDataUtils.toXyzPointList` use the same `forEach` pattern with lambdas. I think this may break the consistency if just rewrite one of them?

Reply:
`forEach` is nice for maps, like here, due to the tedium of entrySet. We should never use it with lists though since normal for is basically the same.